### PR TITLE
applied styling to mobile nav

### PR DESCRIPTION
**Pull request by darushka**
Updated style of mobile anv to match the Spec.

### CODE REVIEW

File: src/directives/sidenav/sidenav.html [hunk 1]
```diff
@@ -50,19 +50,23 @@ <h4 ng-bind="app.name"></h4>
...
     </peach-accordion-panel>
-    <div class="peach-static-panel">
+    <div class="peach-static-panel" ng-if="sidenav.coreAppsBottom.length">
       <md-list>
         <md-list-item ng-repeat="page in sidenav.coreAppsBottom track by $index" ng-class="{'active': sidenav.isPageActive(page.path)}">
           <span class="icon icon-{{page.key}}"></span>
-          <a ng-href="{{page.path}}">{{page.name}}</a>
-        </md-list-item>
+          <h4><a ng-href="{{page.path}}">{{page.name}}</a></h4>
+        </md-lisdt-item>
```
Comment (jsmylnycky) [inline src/directives/sidenav/sidenav.html]: That should be </md-list-item>  Got an extra 'd' in there by accident :)

File: dist/peach.js [hunk 2]
```diff
@@ -3959,6 +3959,8 @@ return /******/ (function(modules) { // webpackBootstrap
 	      _this.$q.all(promises).then(function (appPages) {
 	        _.forEach(apps, function (app, idx) {
 	          _this.mobileApps[idx].pages = _.reject(appPages[idx], { 'hide_in_nav': false });
+	          _this.mobileApps[idx].hasSettingsPage = _.where(_this.mobileApps[idx].pages, { 'is_settings_page': true }).length > 0;
+	          console.log(_this.mobileApps[idx].hasSettingsPage);
```
Comment (againsley) [inline dist/peach.js]: Are we allowing console.logs into production in V2? At some point we'll want to remove them all but might be easier to keep them out now as opposed to trying to cleanup later.

Comment (jsmylnycky) [inline dist/peach.js]: @againsley  Nope, we want to keep console.logs out.  I'll get it removed.  Good eye, I missed it.

**Comment by againsley** [comment]
:+1:

**Comment by jsmylnycky** [comment]
Release: 38e66c4 v0.3.1

**Pull request closed without merging**